x-pack/platform/packages/private/security/ui_components/src/kibana_privilege_table/feature_table.tsx

```diff
@@ -156,7 +156,7 @@ export class FeatureTable extends Component<Props, State> {
             <EuiSpacer size="s" />
             {helpText && (
               <>
-                <EuiCallOut size="s" title={helpText} />
+                <EuiCallOut announceOnMount={false} size="s" title={helpText} />
```

Comment:
Question: Based on the instructions in the PR, this one is in a conditional and set to `false`, but I dont think it necessarily is dependent on user actions (but I guess it sort of is 😅 ). Any advice?

Reply:
Yes! Thank you!